### PR TITLE
NIN-8: Add consistent styling.

### PR DESCRIPTION
**Pull request by shakedlokits**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 120
+
+[{providers, services}/*/node_modules/**]
+insert_final_newline = false
```
Comment (wildeyes) [inline .editorconfig:13]: why is this needed?

**Review by wildeyes — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (shakedlokits) [reply .editorconfig:13]: Peer deependant modules might use asm/wsm/wtc where this behaviour is not popular or required. Also we shouldn't assume on other peoples code (:
